### PR TITLE
[ie/radiko]add Metadata info (performer, duration)

### PR DESCRIPTION
**Pull request by YoshichikaAAA**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

Adds the following information as metadata that can be obtained from radiko.jp for the target program:
- Performers
- Duration

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by pukkandan** [comment]
> * (Make sure to add/update tests)

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:194]: this is fatal, `unified_timestamp` can return `None`. wrap in `try_call`

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:239]: wrap in `try_call` and inline in return

**Review by bashonly — CHANGES_REQUESTED** [review]

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:249]: note to maintainers: after #8917 is merged, this should be `artists` with a list value. until then leave it as `artist` w/ string value

File: yt_dlp/extractor/radiko.py
Comment (YoshichikaAAA) [reply yt_dlp/extractor/radiko.py]: Thank you for your feedback. I fixed it.

File: yt_dlp/extractor/radiko.py
Comment (YoshichikaAAA) [reply yt_dlp/extractor/radiko.py]: Thank you for your feedback. I fixed it.

**Comment by YoshichikaAAA** [comment]
> > * (Make sure to add/update tests)

This site (radiko.jp) provides Japanese radio programs with live streaming and on-demand.
The metadata for live streaming, such as performer, program duration, dynamically changes depending on the time of execution.
Additionally, on-demand streaming is available for only one week after the broadcast.
Furthermore, it is only accessible within Japan, and channels available for viewing are exclusive to each prefecture determined by IP addresses.

In the original source code, the tests specifically target URL detection only.
Considering it will be executed in CI, creating tests beyond the current scope is difficult.

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:38]: the separator character class is a good idea, but we can inline this in the method
```suggestion
```

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:166]: This is still fatal (`try_call` returns `None` when it fails, which is not iterable), and the `artists` PR has not been merged yet, so we'll still need a string for now:
```suggestion
    def _extract_performers(self, prog):
        performers = traverse_obj(prog, (
            'pfm/text()', ..., {lambda x: re.split(r'[/／、　,，]', x)}, ..., {str.strip}))
        # TODO: change 'artist' fields to 'artists' and return traversal list instead of str
        return ', '.join(performers) or None
```

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:195]:
```suggestion
            'artist': self._extract_performers(prog),
```

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:245]:
```suggestion
```

File: yt_dlp/extractor/radiko.py
Comment (bashonly) [inline yt_dlp/extractor/radiko.py:255]:
```suggestion
            'artist': self._extract_performers(prog),
```

**Review by bashonly — APPROVED** [review]

**Review by seproDev — APPROVED** [review]

**Pull request merged**
